src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -56,108 +70,109 @@ private class CacheItem
         };
 
         /// <summary>Default maximum number of transactions in the cache.</summary>
-        public const int CacheMaxItemsDefault = 100000;
+        public const int CacheMaxItemsDefault = 100_000;
+
+        /// <summary>The current batch size in bytes.</summary>
```

Comment:
I think this is the old comment when the code was storing the batch size in bytes.  Now it's doing a count of items.